source/clients.rst

```diff
@@ -3,10 +3,10 @@
 Clients
 -------
 
-The `Oneclient` code and basic documnentation is available on GitHub:
+The `Oneclient` code and basic documentation is available on GitHub:
```

Comment:
surely `oneclient` should be written lower-case if it's in code blocks, and upper-case if it's the name of the tool?